windows_containers/creating_windows_machinesets/creating-windows-machineset-nutanix.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="creating-windows-machineset-nutanix"]
+= Creating a Windows MachineSet object on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: creating-windows-machineset-nutanix
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on Nutanix. For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
+
+[discrete]
+include::modules/machine-api-overview.adoc[leveloffset=+1]
+include::modules/windows-machineset-nutanix.adoc[leveloffset=+1]
+include::modules/machineset-creating.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
+
+* For more information on managing machine sets, see the _Machine management_ section.
```

Comment:
Other cloud providers have an actual xref here. I'd take a look at those assemblies and do what they did, for consistency.

You are allowed to xref here, because this is an assembly, not a module.